packages/grafana-ui/src/components/FileDropzone/FileDropzone.tsx

```diff
@@ -134,9 +135,15 @@ export function FileDropzone({ options, children, readAs, onLoad, fileListRender
   const removeFile = (file: DropzoneFile) => {
     const newFiles = files.filter((f) => file.id !== f.id);
     setFiles(newFiles);
+    onFileRemove && onFileRemove(file);
   };
 
-  const { getRootProps, getInputProps, isDragActive } = useDropzone({ ...options, useFsAccessApi: false, onDrop });
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({
+    ...options,
+    useFsAccessApi: false,
+    onDrop,
+    accept: options?.accept,
```

Comment:
Any reason we need to explicitly pass `accept` here as we're already passing the whole options object?

Reply:
ah good call! fixed.